src/models/Scholarships.test.ts

```diff
@@ -369,23 +404,43 @@ test('scholarships.new - default values', async () => {
 });
 
 test('includesFilter() - no requirements', () => {
```

Comment:
Gonna want to update these names. Also you can use simple array values (instead of `GradeLevel` ones) for these since it's a generic function comparing two arrays.